Xamarin.Essentials/Accelerometer/Accelerometer.shared.cs

```diff
@@ -4,25 +4,23 @@ namespace Xamarin.Essentials
 {
     public static partial class Accelerometer
     {
+        static bool useSyncContext;
+
```

Comment:
This was a property, but I made it a field to avoid confusion with the public properties.